.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "js/polkadot-launch"]
+	path = js/polkadot-launch
+	url = https://github.com/paritytech/polkadot-launch.git
```

Comment:
This is obsolete now, right? since we checkout during build?

Reply:
This still exists QAQ, 

https://github.com/ChainSafe/PINT/pull/83#discussion_r652332446

We could not avoid using the submodule in current state I think, because the CI can not fetch this repo as well

Reply:
I see, guess we'd have to live with it for now.

Reply:
> This is obsolete now, right? since we checkout during build?

OO, This will not load into our directory unless we `git submodule update --init`